src/custom-typings.d.ts

```diff
@@ -43,9 +43,6 @@ import * as _ from 'lodash'
  *
  */
 
-// support NodeJS modules without type definitions
-declare module '*';
```

Comment:
this is fine. I rather have developers who are new to typescript not run into problems when installing 3rd party modules. you can add a comment in the readme about removing this line if they use `noImplicitAny`